components/econet/climate/econet_climate.cpp

```diff
@@ -45,20 +41,17 @@ climate::ClimateTraits EconetClimate::traits() {
   }
   traits.set_supports_current_temperature(true);
   if (this->parent_->get_model_type() == MODEL_TYPE_HVAC) {
-    traits.set_visual_min_temperature(10);
-    traits.set_visual_max_temperature(32);
```

Comment:
The default values are 10 and 30 which are reasonable for HVAC so I removed these. We can override them in yaml anyway.